### ISSUE TITLE
Adding support for for fawry callback to update order status

### ISSUE THREAD

**Issue by omaratta212** [taekunger/Woocommerce-Fawry-Payment-Gateway#1]
That's a very powerful yet simple plugin, just wondering if you could add support for the callback URL sent by Fawry to update the order status in Woocommerce that would be very helpful.

Here is the docs for the callback v1: https://www.atfawry.com/developer-guide/callBackV1.html

**Comment by jadolyo** [comment]
Yes, I would love to see a callback update for this plugin, Any plans or it's dead?
